src/components/Layout/Layout.stories.tsx

```diff
@@ -152,6 +169,52 @@ function TableExample({ numCols = 10, numRows = 100 }: { numCols?: number; numRo
   );
 }
 
+function VirutalizedPage() {
+  const { pl } = useScrollableParent();
+  return (
+    <>
+      <TestHeader title="Change Event - Mud Room" />
+      <p>Content above the table</p>
+      {/*
+        Providing a <ScrollableContent> wrapper here isn't really necessary,
+        as the Virtualized component will provides its own scrolling behavior.
+        Though, because we want to wrap the VirtualizedTable in an element to provide some consistent padding,
+        that means we need to set h100 on the wrapping element. Since setting h100 means it'll take the full height of the parent,
+        then it's better to wrap in ScrollableContent because ScrollableContent's height is based on the remaining space available in the viewport. But...
+
+        Side note: Even removing <ScrollableContent /> here and keeping the wrapping div with `h100` seems to work.
+        I am kind of perplexed on why `h100` isn't taking up 100% of the parent container, but somehow taking up just the remaining space.
+        For example, my parent container is 868px. Within this container I have the <TestHeader /> and <p>, then the `h100` div below.
+        That `h100` was translating to '735px', though if I set the `height: 10%`, then it translates to 86.8px. (if 10% is 86.8 how is 100% 735???)
+        Because I am unsure why/how ends up taking up the correct amount of space, I would push to continue to wrap inside of <ScrollableContent />
+        ¯\_(ツ)_/¯
+      */}
+      <ScrollableContent>
```

Comment:
Wdyt about having like a `ScrollableContent virtualized=true` prop that could probably do all of:

* Add in `FullBleed` with `omitPadding` 
* Look up the current `pl` value
* Add the `h100.pl` wrapper